src/main/java/com/bunq/sdk/context/ApiContext.java

```diff
@@ -230,13 +230,18 @@ private void deleteSession() {
    * needed.
    */
   public void ensureSessionActive() {
-    if (sessionContext == null) {
-      return;
-    }
+   if (!isSessionActive()){
+     resetSession();
+   }
+  }
+
+  /**
+   * @return True if session has expired, otherwise false.
```

Comment:
Please remove this comment. This method is self-descriptive enough, and also the comment is not very precise :P